src/npc.c

```diff
@@ -13,6 +13,12 @@ extern s16 D_80077C1C;
 extern s16 D_80077C1E;
 extern s32 D_80077C20;
 
+s32 playFX_0C(f32, f32, f32, f32, f32); // extern
+extern u16 D_80077C30;
+extern s32 D_80077C34;
+s32 playFX_23(u32, f32, f32, f32, f32); // extern
```

Comment:
I would move the prototypes for these functions to `include/functions.h` for now. There's a big section on line 390 marked 'Effect funcs' where they can live.